modules/heat_conduction/test/tests/restart/restart.i

```diff
@@ -0,0 +1,162 @@
+[GlobalParams]
+  order = SECOND
+  family = LAGRANGE
+[]
+
+[Problem]
+  coord_type = RZ
+[]
+
+[Mesh]
+  file = restart.e
+  patch_size = 10 # 1000 # For contact algorithm
+[]
+
+[Variables]
+  [./temp]
+    initial_condition = 580.0     # set initial temp to coolant inlet
+  [../]
+[]
+
+[AuxVariables]
+  [./gap_cond]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+[]
+
+[Kernels]
+  [./heat]
+    type = HeatConduction
+    variable = temp
+  [../]
+  [./heat_source]
+    type = BodyForce
+    variable = temp
+    block = pellet_type_1
+    value = 1e-2
+    function = 't'
+  [../]
+[]
+
+[ThermalContact]
+  [./thermal_contact]
+    type = GapHeatTransfer
+    variable = temp
+    master = 5
+    slave = 10
+    quadrature = true
+  [../]
+[]
+
+[BCs]
+  [./cladboundary]
+    type = DirichletBC
+    value = 580
+    boundary = '1 2 3'
+    variable = temp
+  [../]
+  [./fuelboundary]
+    type = PresetBC
+    value = 700
+    boundary = 10
+    variable = temp
+  [../]
+[]
+
+[Materials]
+  [./fuel_thermal]
+    type = HeatConductionMaterial
+    block = pellet_type_1
+    thermal_conductivity = 5
+    specific_heat = 12
+  [../]
+  [./clad_thermal]
+    type = HeatConductionMaterial
+    block = clad
+    thermal_conductivity = 16.0
+    specific_heat = 330.0
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = 'PJFNK'
+
+  petsc_options = '-snes_ksp_ew'
+  petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
+  petsc_options_value = ' lu       superlu_dist'
+  line_search = 'none'
+
+  l_max_its = 100
+
+  nl_max_its = 50
+  nl_rel_tol = 1e-4
+  nl_abs_tol = 1e-8
+
+  start_time = -200
+  n_startup_steps = 1
+  end_time = 1.02e5
+  num_steps = 10
+
+  dtmax = 2e6
+  dtmin = 1
+
+  [./TimeStepper]
+    type = IterationAdaptiveDT
+    dt = 2.0e2
+    optimal_iterations = 15
+    iteration_window = 2
+  [../]
+
+  [./Quadrature]
+    order = FIFTH
+    side_order = SEVENTH
+  [../]
+[]
+
+[Postprocessors]
+  [./ave_temp_interior]
```

Comment:
Probably no point keeping any of these -- let's make this as simple as possible.

Reply:
These I'd rather keep as they are good checks that the restart occurred properly.